### DISCUSSION TITLE
Sort by number of pages

### DISCUSSION THREAD

**Discussion by OncePunchedMan** [gotson/komga#1570] [Ideas]
### Describe your suggested feature

Currently there is no way to sort by number of pages.
However Komga detects how many pages there are already, so sorting by it might be a nice addition imo.

### Other details

Basically the same as #734. It got closed automatically back then and i wondered if it could be thought about some more?

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by Blazeflack** [comment]
Commenting to show my support of this suggestion.

**Comment by gotson** [comment]
🎉 This issue has been resolved in `1.16.0` ([Release Notes](https://github.com/gotson/komga/releases/tag/1.16.0))
